app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityPresenter.kt

```diff
@@ -86,10 +91,47 @@ class AdministratorControlsActivityPresenter @Inject constructor(
     getAdministratorControlsFragment()!!.setSelectedFragment(lastLoadedFragment)
     activity.supportFragmentManager.beginTransaction().add(
       R.id.administrator_controls_fragment_multipane_placeholder,
-      AppVersionFragment()
+      AppVersionFragment(),
+      ADMINISTRATOR_CONTROLS_BACKSTACK
     ).commitNow()
   }
 
+  fun loadProfileEdit(profileId: Int) {
+    lastLoadedFragment = PROFILE_EDIT_FRAGMENT
+    binding.administratorControlsMultipaneOptionsBackButton!!.visibility = View.VISIBLE
+    val fragment = ProfileEditFragment.newInstance(profileId, isMultipane)
+    activity.supportFragmentManager.beginTransaction().add(
+      R.id.administrator_controls_fragment_multipane_placeholder,
+      fragment,
+      ADMINISTRATOR_CONTROLS_BACKSTACK
+    )
+      .addToBackStack(ADMINISTRATOR_CONTROLS_BACKSTACK)
+      .commit()
```

Comment:
We should always use commitNow().

Reply:
error comes if I use commitNow

2022-01-09 23:30:26.063 4994-4994/org.oppia.android E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.oppia.android, PID: 4994
    java.lang.IllegalStateException: This transaction is already being added to the back stack
        at androidx.fragment.app.FragmentTransaction.disallowAddToBackStack(FragmentTransaction.java:679)
        at androidx.fragment.app.BackStackRecord.commitNow(BackStackRecord.java:296)
        at org.oppia.android.app.administratorcontrols.AdministratorControlsActivityPresenter.loadProfileEdit(AdministratorControlsActivityPresenter.kt:119)
        at org.oppia.android.app.administratorcontrols.AdministratorControlsActivity.loadProfileEdit(AdministratorControlsActivity.kt:104)
        at org.oppia.android.app.settings.profile.ProfileListFragmentPresenter$bindProfileView$1.onClick(ProfileListFragmentPresenter.kt:76)
        at android.view.View.performClick(View.java:6597)
        at android.view.View.performClickInternal(View.java:6574)
        at android.view.View.access$3100(View.java:778)
        at android.view.View$PerformClick.run(View.java:25885)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)

Reply:
No longer applies since we're now using the fragment approach used elsewhere in the app.